CODEOWNERS

```diff
@@ -10,18 +10,18 @@
 # Dependencies
 /go.mod @johanstokking @htdvisser
 /go.sum @johanstokking @htdvisser
-/package.json @bafonins @kschiffer @ryaplots
-/yarn.lock @bafonins @kschiffer @ryaplots
-/sdk/js/package.json @bafonins @kschiffer
-/sdk/js/yarn.lock @bafonins @kschiffer
+/package.json @kschiffer @ryaplots
+/yarn.lock @kschiffer @ryaplots
+/sdk/js/package.json @kschiffer
+/sdk/js/yarn.lock @kschiffer
```

Comment:
Add @ryaplots here too, and in all other places where @kschiffer would be the only code owner. We always need at least two so things don't block for too long.